warehouse/utils/webauthn.py

```diff
@@ -141,21 +154,26 @@ def verify_assertion_response(assertion, *, challenge, user, origin, rp_id):
     Returns an updated signage count on success.
     Raises AuthenticationRejectedError on failure.
```

Comment:
```suggestion
    Raises InvalidAuthenticationResponse on failure.
```
There's some other places this should be updated.

Reply:
no, we capture the `InvalidAuthenticationResponse` from pywebauthn, the re-raise our own exception called `AuthenticationRejectedError`